helixbass/line-runner
# Ahead/behind the beat configuration

In this PR:
- fire note on's asynchronously
- allow configuring mapping a MIDI slider to behind/ahead of the beat ratio

To test:
If per the updated README you add an `ahead_or_behind_the_beat_ratio_slider` MIDI slider configuration object to the `midi` config section and run, you should be able to make the notes of the line get played anywhere from a full sixteenth note ahead of the beat (slider all the way to the left) to a full sixteenth note behind the beat (slider all the way to the right)
